src/codex_plus/llm_execution_middleware.py

```
@@ -363,61 +444,356 @@ async def process_request(self, request, path: str):
+                        raise
+                    finally:
+                        if not response_closed:
+                            try:
+                                response.close()
+                            except Exception:
+                                pass
+                        if hasattr(response, "_close_cm"):
+                            try:
+                                response._close_cm(None, None, None)
+                            except Exception:
+                                pass
```

Ruff (0.13.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)